recipes/centos_spring_boot/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+set -e
+
+# insert settings for mirrors/repository managers into settings.xml if supplied
+function configure_mirrors {
+  if [ -n "$MAVEN_MIRROR_URL" ]; then
+    xml="    <mirror>\
+      <id>mirror.default</id>\
+      <url>$MAVEN_MIRROR_URL</url>\
+      <mirrorOf>external:*</mirrorOf>\
+    </mirror>"
+    sed -i "s|<!-- ### configured mirrors ### -->|$xml|" $HOME/.m2/settings.xml
+  fi
+}
+
+# check for the MAVEN_MIRROR_URL env variable, if its available then set maven mirrors in $HOME/.m2/settings.xml
+configure_mirrors
```

Comment:
missing new line

Also please check on https://www.shellcheck.net/

Reply:
Thanks for the site, has been using yamllint/jsonlint for longtime never knew we have tool like this very handy